src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -53,14 +53,33 @@ public function getMaxDepth();
 
     /**
      * Sets the serialization name for this attribute.
+     *
+     * @deprecated since Symfony 5.2, use addSerializedName instead.
```

Comment:
I'm not sure if this is useful because this class is internal.

Reply:
not sure either, your call

Reply:
@nicolas-grekas what do you think?